LibSWBF2.NET/Wrappers/Model.cs

```diff
@@ -15,6 +15,8 @@ internal Model(IntPtr modelPtr) : base(modelPtr)
 
         }
 
+        public Model() : base(IntPtr.Zero){}
+
```

Comment:
The ```new()``` constraint required in ```IntPtrsToWrappers``` requires a public parameterless constructor.  I figured this could be problematic since it enables users to create invalid objects/might be confusing...